### ISSUE TITLE
Execution question

### ISSUE THREAD

**Issue by pantinor** [verils/gotemplate4j#4]
Is there a way to execute the template with provided yaml such that the template can parse golang ".Values" from the input values yaml?

{{- if .Values.config.local -}}
    {{- range $key, $val := .Values.config.local -}}

and the input is just the yaml like:

config:
  local:
    etc:

**Comment by pantinor** [comment]
you may be already aware that this above templating syntax that referencing **.Values** is how helm templating works.

**Comment by verils** [comment]
No. This project is an implementation of golang's standard text/template in Java, the Helm features are not in consideration. There should be another project to do these things, I don't have any planning for such a project.

**Comment by pantinor** [comment]
Thanks.  Do you know if there are any java libraries that can emulate helm templating implementation?

**Comment by verils** [comment]
I haven't found one yet.
